input/pagecontent/export.md

```diff
@@ -23,37 +23,53 @@ The scope of this document does NOT include:
 
-Data access control obligations can be met with a combination of in-band restrictions such as OAuth scopes, and out-of-band restrictions, where servers limit the data returned to a specific client in accordance with local considerations (e.g.  policies or regulations). For example, some clients are authorized to access sensitive mental health information and some aren't; this authorization is defined out-of-band, but when a client requests a full data set, filtering is automatically applied by the server, restricting the data that the client receives.
+Data access control obligations can be met with a combination of in-band restrictions such as OAuth scopes, and out-of-band restrictions, where servers limit the data returned to a specific client in accordance with local considerations (e.g.  policies or regulations). The FHIR server SHALL limit the data returned to only those FHIR resources for which the client is authorized. Implementers SHOULD incorporate technology that preserves and respects an individual's wishes to share their data with desired privacy protections. For example, some clients are authorized to access sensitive mental health information and some aren't; this authorization is defined out-of-band, but when a client requests a full data set, filtering is automatically applied by the server, restricting the data that the client receives.
 
-Bulk data export can be a resource-intensive operation. Server developers should consider and mitigate the risk of intentional or inadvertent denial-of-service attacks (though the details are beyond the scope of this specification).
+Bulk data export can be a resource-intensive operation. Server developers should consider and mitigate the risk of intentional or inadvertent denial-of-service attacks (though the details are beyond the scope of this specification). For example, transactional systems may wish to provide Bulk Data access to a read-only mirror of the database, or may distribute processing over time to avoid loads that could impact clinical operations.
 
 ### Request Flow
 
+This implementation guide builds on the [FHIR Asynchronous Request Pattern](http://hl7.org/fhir/R4/async.html), and in some places may extend the pattern.
+
+#### Roles
+
+There are two primary roles involved in a bulk data transaction:
+
+  1. **Bulk Data Provider** - system that provides bulk data. Consists of:
+
+      a. **Authorization Server** - issues access tokens in response to valid token requests from client
```

Comment:
Comparing this with language tweaks we made in the Backend Services pages, should this say "FHIR Authorization Server"? (Or... why the difference?)